internal/controller/vspherevalidator_controller.go

```diff
@@ -188,13 +189,13 @@ func (r *VsphereValidatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		if err != nil {
 			r.Log.V(0).Error(err, "failed to reconcile computeresources validation rule")
 		}
-		vres.SafeUpdateValidationResult(r.Client, nn, validationResult, failed, err, r.Log)
+		vres.SafeUpdateValidationResult(r.Client, nn, validationResult, err, r.Log)
 		r.Log.V(0).Info("Validated compute resources", "scope", rule.Scope, "entity name", rule.EntityName)
 	}
 
 	// requeue after two minutes for re-validation
 	r.Log.V(0).Info("Requeuing for re-validation in two minutes.", "name", req.Name, "namespace", req.Namespace)
-	return ctrl.Result{RequeueAfter: 2 * time.Minute}, nil
```

Comment:
💬 Why did you remove the `RequeueAfter`? That's the standard in all the other plugins.